### ISSUE TITLE
Alias creation

### ISSUE THREAD

**Issue by johnp1954** [syscoin/blockmarket-desktop-public#363]
User Altcoinlover had been having problems creating Aliases, it turns out she was closing BM before the process had completed. There is no real information telling users to wait until the Wallet Activity has finished. Can the big message, bottom right, that says alias reservation stay until the whole process is complete

**Comment by jnessbcf** [comment]
Yeah I made a note of that too; seemed like there were a couple of people who assumed (and for good reason - there’s no other notification!) that they should reboot to see their alias.
